scripts/deploy-staging.sh

```diff
@@ -0,0 +1,17 @@
+# !/usr/bin/bash
+
+set -e -x
+
+BRANCH=$(git symbolic-ref --short HEAD)
+APP_NAME=ervell-staging
+S3_BUCKET=ervell-staging
```

Comment:
I think you can just export these. Don't inline the other scripts. They might (will) change